packages/components/src/mobile/gradient/index.native.js

```diff
@@ -111,17 +121,13 @@ function Gradient( {
 
 	const colorGroup = getGradientColorGroup( gradientValue );
 
-	const locations = colorGroup.map(
-		( location ) => Number( location[ 1 ].replace( '%', '' ) ) / 100
-	);
-
 	if ( isLinearGradient ) {
 		return (
 			<RNLinearGradient
 				colors={ getGradientBaseColors( gradientValue ) }
 				useAngle={ true }
 				angle={ getGradientAngle( gradientValue ) }
-				locations={ locations }
+				locations={ getColorLocations( gradientValue ) }
```

Comment:
The `getColorLocations` always returns a new reference. Maybe we could use the `useMemo` here?

Reply:
Done, please review it one more time :v: